Makefile

```diff
@@ -23,7 +23,7 @@ MSBUILD_LIBRARIES=Microsoft.Build.dll Microsoft.Build.Framework.dll Microsoft.Bu
 MSBUILD_DLLS=$(patsubst %, $(MSBUILD_PATH)/%, $(MSBUILD_LIBRARIES))
 
 # Set $PATH to point to provisioned .NET Core and avoid the ones provisioned by VSTS itself
-all: export PATH:="/usr/local/share/dotnet:$(PATH)"
+all: export PATH="/usr/local/share/dotnet:$PATH"
```

Comment:
why is this change needed? the commit message doesn't say the reason for it

Reply:
@knocte The makefile change was made to let Ubuntu 16.04/18.04 compilation pass. If reverted, then those two platforms fail (https://github.com/lextm/dotdevelop/actions/runs/462099248).

Reply:
then add this rationale in the commit message pls